contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -140,6 +140,39 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
         balance = balance.scaleBy(assetDecimals, 18) / THREEPOOL_ASSET_COUNT;
     }
 
+    /**
+     * @dev This function is completely analogous to _calcCurveTokenAmount[BaseCurveStrategy]
+     * and just utilizes different Curve (meta)pool API
+     */
+    function _calcCurveMetaTokenAmount(uint128 _coinIndex, uint256 _amount)
+        internal
+        returns (uint256 requiredMetapoolLP)
+    {
+        uint256[2] memory _amounts = [uint256(0), uint256(0)];
+        _amounts[uint256(_coinIndex)] = _amount;
+
+        // LP required when removing required asset ignoring fees
+        uint256 lpRequiredNoFees = metapool.calc_token_amount(_amounts, false);
+        // LP required if fees would apply to entirety of removed amount
+        uint256 lpRequiredFullFees = lpRequiredNoFees.mulTruncateScale(
```

Comment:
Same comment about `mulTruncateScale` expecting three params but given only two here as above

Reply:
The first parameter is the `lpRequiredNoFees`